test/message/test_plan.py

```diff
@@ -449,10 +530,9 @@ def plan_mult_3(data):
     # Execute it locally
     x = th.tensor([-1, 2, 3])
     assert (get_plan(x) == th.tensor([-3, 6, 9])).all()
-    assert fetched_plan.is_built
     assert (fetched_plan(x) == th.tensor([-3, 6, 9])).all()
-
-    hook.local_worker.is_client_worker = True
```

Comment:
Keep this line
when we call hook.local_worker.is_client_worker = False in a test we should **always** revert it at the end. 
Otherwise, I suffer a lot :D

Reply:
nice catch, thanks!